workos/utils/request.py

```diff
@@ -67,10 +67,16 @@ def request(
             response = request_fn(url, headers=headers, json=params)
 
         response_json = None
-        try:
-            response_json = response.json()
-        except ValueError:
-            raise ServerException(response)
+        content_type = (
+            response.headers.get("content-type")
+            if response.headers is not None
```

Comment:
This null check is reasonable, but do we know when `response.headers` is possibly `None`?

I'm not familiar with the API, so I could see it possibly always defaulting to an empty dictionary. But maybe it is `None` in the case of client errors when there is no response at all?

Reply:
I think I added this check after I had seen it in my test cases. Not sure if it is because we have a specific instance where we're setting the headers to `None`.